Sources/Misc/SystemInfo.swift

```diff
@@ -57,8 +57,12 @@ class SystemInfo {
     private static let _forceUniversalAppStore: Atomic<Bool> = false
     private static let _proxyURL: Atomic<URL?> = nil
 
-    var isSandbox: Bool {
+    private lazy var _isSandbox: Bool = {
         return self.sandboxEnvironmentDetector.isSandbox
+    }()
```

Comment:
Caching the result since the sandbox detector is a bit more CPU-intensive, and does not change.

Reply:
I was worried this was suddenly a lot less efficient, but:
- It's macOS only
- It's cached 👏🏻